packages/agent-sdk-ts/docs/python-parity.md

```
@@ -0,0 +1,273 @@
+      +startNewConversation()
+      +restoreConversation()
+      +sendUserMessage()
+      +disconnect()/reconnect()
+      -WebSocket
+      -historyReplay()
+    }
+```
+
+## Agent lifecycle and orchestration
+
+**Python shape**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)